db/migrate/20160208173646_add_the_visit_id_to_models.rb

```diff
@@ -0,0 +1,19 @@
+class AddTheVisitIdToModels < ActiveRecord::Migration
+  def up
+    add_column :proposals, :visit_id, :uuid
+    add_column :comments, :visit_id, :uuid
+    add_column :reports, :visit_id, :uuid
+    add_foreign_key :proposals, :visits
+    add_foreign_key :comments, :visits
+    add_foreign_key :reports, :visits
+  end
+
+  def down
+    remove_foreign_key :proposals, :visits
+    remove_foreign_key :comments, :visits
+    remove_foreign_key :reports, :visits
+    drop_column :proposals, :visit_id
```

Comment:
good catch! we should make sure we are running and rolling back each migration while they are in developing. ensures we have migrations with proper `up` and `down` content